userland/examples/ble_adv/ble.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <tock.h>
+
+/*------- SYSCALLS------------*/
+#define DRIVER_RADIO      33
+
+// commands calls
+#define BLE_ADV_START     0
+#define BLE_ADV_STOP      1
+#define CFG_TX_POWER      2
+#define CFG_ADV_INTERVAL  3
+/*----------------------------*/
+
+// AD Types
+#define BLE_HS_ADV_TYPE_FLAGS                   0x01
+#define BLE_HS_ADV_TYPE_INCOMP_UUIDS16          0x02
+#define BLE_HS_ADV_TYPE_COMP_UUIDS16            0x03
+#define BLE_HS_ADV_TYPE_INCOMP_UUIDS32          0x04
+#define BLE_HS_ADV_TYPE_COMP_UUIDS32            0x05
+#define BLE_HS_ADV_TYPE_INCOMP_UUIDS128         0x06
+#define BLE_HS_ADV_TYPE_COMP_UUIDS128           0x07
+#define BLE_HS_ADV_TYPE_INCOMP_NAME             0x08
+#define BLE_HS_ADV_TYPE_COMP_NAME               0x09
+#define BLE_HS_ADV_TYPE_TX_PWR_LVL              0x0a
+#define BLE_HS_ADV_TYPE_SLAVE_ITVL_RANGE        0x12
+#define BLE_HS_ADV_TYPE_SOL_UUIDS16             0x14
+#define BLE_HS_ADV_TYPE_SOL_UUIDS128            0x15
+#define BLE_HS_ADV_TYPE_SVC_DATA_UUID16         0x16
+#define BLE_HS_ADV_TYPE_PUBLIC_TGT_ADDR         0x17
+#define BLE_HS_ADV_TYPE_RANDOM_TGT_ADDR         0x18
+#define BLE_HS_ADV_TYPE_APPEARANCE              0x19
+#define BLE_HS_ADV_TYPE_ADV_ITVL                0x1a
+#define BLE_HS_ADV_TYPE_SVC_DATA_UUID32         0x20
+#define BLE_HS_ADV_TYPE_SVC_DATA_UUID128        0x21
+#define BLE_HS_ADV_TYPE_URI                     0x24
+#define BLE_HS_ADV_TYPE_MFG_DATA                0xff
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int ble_adv_data(uint8_t type, uint8_t len, const unsigned char *data);
+int ble_adv_set_txpower(uint8_t);
```

Comment:
Cool, I see the defines above now. However, currently `ble_adv_set_txpower` takes a uint8_t. You should probably change that to an `int8_t`.